### PR TITLE
Fix/condo/doma 5570/server side api bug

### PR DESCRIPTION
**Pull request by AleX83Xpert**
Some logs were added to detect error point. To reach this goal the request object passed to providers and plugins to check reqId at logs.

### CODE REVIEW

File: apps/address-service/domains/common/utils/services/search/plugins/SearchByFiasId.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ class SearchByFiasId extends AbstractSearchPlugin {
      */
     isEnabled (s, params) {
```
Comment (akarjakin) [inline apps/address-service/domains/common/utils/services/search/plugins/SearchByFiasId.js:22]: if `this` is not used within method, than it's a static method actually.

File: apps/address-service/domains/common/utils/services/search/plugins/SearchByGooglePlaceId.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ class SearchByGooglePlaceId extends AbstractSearchPlugin {
      */
     isEnabled (s, params) {
```
Comment (akarjakin) [inline apps/address-service/domains/common/utils/services/search/plugins/SearchByGooglePlaceId.js:21]: same

File: apps/address-service/domains/common/utils/services/search/plugins/SearchByProvider.js [hunk 3]
```diff
@@ -15,15 +15,15 @@ class SearchByProvider extends AbstractSearchPlugin {
      * @returns {boolean}
      */
     isEnabled (s, params) {
```
Comment (akarjakin) [inline apps/address-service/domains/common/utils/services/search/plugins/SearchByProvider.js:17]: same

File: apps/address-service/index.js [hunk 4]
```diff
@@ -98,6 +98,8 @@ module.exports = {
     ].filter(identity),
     configureExpress: (app) => {
         app.set('trust proxy', 1) // trust first proxy
+        app.use(bodyParser.json({ limit: '100mb', extended: true }))
```
Comment (akarjakin) [inline apps/address-service/index.js:101]: can you please mvoe limit to a constant

**Review by akarjakin — APPROVED** [review]

**Review by Alllex202 — APPROVED** [review]

**Pull request merged**